boards/px4/fmu-v5/init/rc.board_defaults

```diff
@@ -23,6 +23,9 @@ then
 	param set UAVCAN_ENABLE 2
 fi
 
+# Use always FMU as RC input
+param set SYS_RC_SOURCE 2
```

Comment:
Forcing the value every boot would be kind of frustrating if a user actually wanted to configure it (for some reason) or entirely disable it.

Is the intent for it to actually be user configurable or do you want to only operate this way?